hooks/fetchTarotDeck.ts

```diff
@@ -27,8 +27,9 @@ function useFetchTarotDeck(): TarotDeckProps {
         setLoading(true);
         const { data, error } = await supabase
           .from('tarotCardInformation')
-          .select('*');
-
+          .select('*')
```

Comment:
**question:** Did we need the * for this call?

Reply:
Seems to be fine without, removed!